docs/core/diagnostics/dumps.md

```diff
@@ -10,7 +10,7 @@ A dump is a file that contains a snapshot of the process at the time it was crea
 
 ## Collect dumps
 
-Dumps can be collected in a variety of ways depending on which platform you are running your app on.
+Dumps can be collected in a variety of ways depending which platform you are running your app on.
```

Comment:
Hmm, I actually think the original version is correct. While the "on" is duplicated, it has different meanings in each place. "depending on [which platform]" and "running your app on [a platform]". @tdykstra do you agree?

Reply:
Agree